runnables/binning_dim2.py

```diff
@@ -0,0 +1,111 @@
+import argparse
```

Comment:
File name should be llike `two_dim_binning.py`? Means two dimensional binging.

Reply:
Yes, the original name is two_dim_binning.py. And then changed to binning_dim2.py, it's shorter and the prefix is aligned with binning.py.

Reply:
`two_dim_binning.py` is more explicit, will change to it.